client/src/js/components/bhReportPreview.js

```diff
@@ -0,0 +1,20 @@
+angular.module('bhima.components')
+.component('bhReportPreview', {
+  bindings : {
+    sourceDocument : '<',
+    onClearCallback : '&',
```

Comment:
It would be useful to put a comment here saying that these methods are called in the template - I was confused for a second.